src/main/java/uk/gov/justice/digital/delius/jpa/standard/entity/Contact.java

```diff
@@ -187,6 +193,10 @@
     @Column(name = "RAR_ACTIVITY")
     private String rarActivity;
 
+    @OneToMany(cascade = CascadeType.ALL, mappedBy = "contact", orphanRemoval = true)
+    @Default
+    private List<Enforcement> enforcements = new ArrayList<>();
```

Comment:
I presume the default here is a copy-paste from delius api? I dont think we need it here as that was about nullability in kotlin & defaults when creating new contacts

Reply:
Fixed